new.rst

```diff
@@ -0,0 +1,97 @@
+.. _whats_new:
+
+###############################
+What's New in {Singularity} 4.0
+###############################
+
+This section highlights important changes in {Singularity} 4.0 that are of note
+to users. See also the "What's New" section in the Admin Guide for
+administrator-facing changes.
+
+********
+OCI-mode
+********
+
+Singularity 4 introduces :ref:`OCI-mode <oci_mode>` as a fully supported
+feature. It is enabled by using the ``--oci`` flag with the run / shell / exec /
+pull commands, or when ``oci mode = yes`` is set in ``singularity.conf``.
+
+In OCI-mode:
+
+- Container images from OCI sources will be pull-ed to an :ref:`OCI-SIF
```

Comment:
"pull-ed" looks sort of weird here... Maybe change to "`pull`-ed" or "pulled"?

Reply:
Ahh yep, it was meant to be `pull`-ed. Forgot to add back the formatting after copy and pasting.